client/my-sites/importer/importer-medium.jsx

```diff
@@ -29,12 +30,25 @@ class ImporterMedium extends React.PureComponent {
 		);
```

Comment:
Is it worth moving "Medium" to a translation `arg` here, as well?

Reply:
Look like this was resolved.